tests/hazmat/backends/test_openssl.py

```diff
@@ -529,7 +534,7 @@ def test_checks_for_unsupported_extensions(self):
         ).not_valid_after(
             datetime.datetime(2020, 1, 1)
         ).add_extension(
-            x509.IssuerAlternativeName([x509.DNSName(u"crypto.io")]), False
+            DummyExtension(), False
```

Comment:
I think this is the only thing that hits the KeyError condition in the certbuilder? It probably belongs in test_x509 though and not the openssl specific backend section.